.github/workflows/notebooks.yaml

```diff
@@ -0,0 +1,47 @@
+name: Notebooks
+
+on:
+  # Trigger the workflow on push or pull request, but only on dev branch
+  push:
+    branches:
+      - revamp-2025
+  pull_request:
+    branches:
+      - revamp-2025
+
+permissions:
+  contents: read  # to fetch code
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  documentation:
+    name: Set up Python ${{ matrix.python-version }}
```

Comment:
This is the string that appears in the github actions list; let's make it something like "Notebook integration tests"